### PR TITLE
Add Maths / Sigmoid Function

### PR DESCRIPTION
**Pull request by sukyung99**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: maths/sigmoid.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+After through Sigmoid, the element of the vector mostly 0 between 1. or 1 between -1.
+
+Script inspired from its corresponding Wikipedia article
+https://en.wikipedia.org/wiki/Sigmoid_function
+"""
+
+import numpy as np
+
+
+def sigmoid(vector: float):
```
Comment (xcodz-dot) [inline maths/sigmoid.py:14]:
```suggestion
def sigmoid(vector: np.array):
```

Comment (sukyung99) [reply maths/sigmoid.py]: @xcodz-dot Thank you for your advice. I revised the code and uploaded it.

Comment (xcodz-dot) [reply maths/sigmoid.py]: 😄 thanks

**Review by xcodz-dot — APPROVED** [review]

**Comment by sukyung99** [comment]
@cclauss Can you help me, please?

File: maths/sigmoid.py [hunk 2]
```diff
@@ -36,5 +36,4 @@ def sigmoid(vector: np.array) -> np.array:
 
 if __name__ == "__main__":
     import doctest
-
```
Comment (cclauss) [inline maths/sigmoid.py]: Please run psf/black on this code as discussed in CONTRIBUTING.md because it will reverse this change.

Comment (sukyung99) [reply maths/sigmoid.py]: @cclauss Thank you. I modified the code.

**Review by cclauss — APPROVED** [review]

**Comment by xcodz-dot** [comment]
@dhruvmanila, This seems good to go

**Pull request merged**